styles/Home.module.css

```diff
@@ -1,94 +1,96 @@
 .main {
```

Comment:
Tidied up the styles a little and added in some avatar icons for the participants.